test/Polytopes/types.jl

```diff
@@ -0,0 +1,83 @@
+@testset "types" begin
+
+    a = [1, 2, 3]
+    b = [8, 6, 4]
+
+    for T in [PointVector, RayVector]
+
+        @testset "$T" begin
+
+            for U in [Int64, Polymake.Integer, Polymake.Rational, Float64]
+
+                @testset "$T{$U}" begin
+
+                    @test T{U} <: AbstractVector
+                    @test T{U} <: AbstractVector{U}
+
+                    @test T{U}(a) isa T
```

Comment:
Again: redundant with the next line? Anything `T{U}` is automatically also a `T`, isn't it?

Reply:
Yes, one implies the other so a success covers everything necessary. But a failed `T{U}` test does not always mean a failed `T` test. I usually split this test up so I can faster find the issue when something breaks at basic construction or templating.